app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".KeepingUpWithSchoolActivity">
```

Comment:
Refactored name according to what best describes what the activity is performing.